airbyte-integrations/connectors/source-s3/integration_tests/cloud_spec.json

```diff
@@ -285,7 +285,7 @@
                   "required": ["filetype"]
                 },
                 {
-                  "title": "Document File Type Format (Experimental)",
+                  "title": "Unstructured Document Format",
```

Comment:
These changes are unrelated to 2.0 - they come from upgrading through CDK version 1.3.1.